specifyweb/frontend/js_src/lib/components/FormFields/QueryComboBox.tsx

```diff
@@ -341,7 +341,7 @@ export function QueryComboBox({
     hasTablePermission(field.relatedModel.name, 'create');
 
   return (
-    <div className="flex w-full items-center">
+    <div className="flex w-full min-w-[theme(spacing.40)] items-center">
```

Comment:
Might be worth moving this line to inside of <AutoComplete> so that this change is applied in all the places were autocomplete is used

Reply:
Since you added `min-w-[theme(spacing.20)]` to `<AutoComplete>`, you might be able to revert the change in this file (as it is no longer needed here)